third_party/opa/data-and-policies/user_policies.rego

```diff
@@ -0,0 +1,57 @@
+package data_policies
```

Comment:
is this an example policy rego file which uses the code restructure ? is so, maybe we can provide example of allow and deny enforcement action for completeness?

Reply:
also better to mention somewhere that we can use this template to create new rego file based on new policies.

Reply:
also if this file is an example policy file won't it be better to keep it in a separate directory ? that way it distinguishes between the library functions and the user created rego files?

Reply:
Added example of Deny. Created another subfolder for infrastructure policies. Added another package name for infrastructure policies.